src/pages/product-details/operations/product-details.queries.js

```diff
@@ -51,6 +53,7 @@ export const getProductById = gql`
         }
         bottomMaterial {
           material {
+            translations_key
```

Comment:
we use Camel Case instead of Snake Case

Reply:
of course, but if others have used this method to add a key , I think I should do it to avoid confusion in the code

Reply:
then it makes sense to wait for the hotfix